src/rSharp.cpp

```diff
@@ -50,10 +53,30 @@ void rSharp_create_domain() {
 	}
 
 	// STEP 2: Initialize and start the .NET Core runtime
-	const string_t config_path = STR("./RSharp.runtimeconfig.json");
 	load_assembly_and_get_function_pointer = nullptr;
-	load_assembly_and_get_function_pointer = get_dotnet_load_assembly(config_path.c_str());
+	load_assembly_and_get_function_pointer = get_dotnet_load_assembly(wideStringPath);
 	assert(load_assembly_and_get_function_pointer != nullptr && "Failure: get_dotnet_load_assembly()");
+
+	delete[] wideStringPath;
+}
+
+char_t* MergeLibPath(char** libPath, char* additionalPath)

```

Comment:
Make both arguments into wchar_t instead of char. This method will only concatenate the stuff, not widen the characters and concatenate.